rcongui/package.json

```diff
@@ -50,6 +52,7 @@
     "immutable": "^5.0.3",
     "localforage": "^1.10.0",
     "lodash": "^4.17.21",
+    "miragejs": "^0.1.48",
```

Comment:
shouldn't this be a dev dependency as well?

Reply:
Move to dev dependencies